library/ziti.c

```diff
@@ -390,7 +369,7 @@ int ziti_conn_init(ziti_context ztx, ziti_connection *conn, void *data) {
     c->ziti_ctx = ztx;
     c->data = data;
     c->channel = NULL;
-    c->state = Initial;
+//    c->state = Initial;
```

Comment:
when i see a stray comment like this i wonder if it was intentional. drawing your eyes here just to check